packages/styles/scss/patterns/sub-patterns/content-block/_content-block.scss

```diff
@@ -11,72 +11,71 @@
 @import '../../blocks/feature-card/feature-card';
 
 @mixin content-block {
+  .#{$prefix}--content-block-inverse {
```

Comment:
for modifiers, would usually be double dash: `#{$prefix}--content-block--inverse`.

Reply:
will do

Reply:
Just a sanity check, should it be `inverse`, or should we be naming based on the theme names? cc: @annawen1 @kennylam

Reply:
That's correct. It should be `g10`, `g90`, or `g100`.

Reply:
Yeah i believe it should be inverse